### PR TITLE
Updating starter-cluster to Terraform 0.14

### PR DESCRIPTION
**Pull request by benarent**
I tried running the starter example with Terraform 0.12 but was running into a AWS provider issue with S3.

This bumps the starter example to 0.14.  I'll come back with a follow up after I test the HA version.

### CODE REVIEW

**Review by webvictim — APPROVED** [review]
Looks OK

**Comment by webvictim** [comment]
I have a branch somewhere which updates `ha-autoscale-cluster` to 0.14 as well, it has a lot of other changes in too though so might be better done as a smaller change set.

**Review by klizhentas — APPROVED** [review]

**Comment by benarent** [comment]
@russjones @r0mant This PR needs code owner approval

**Pull request merged**
